src/main/java/frc/robot/command/climb/ZeroClimbersCommand.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.command.climb;
+
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.component.Climber;
+import frc.robot.subsystem.ClimbSubsystem;
+import frc.robot.util.MoPrefs;
+
+public class ZeroClimbersCommand extends Command {
+    private final ClimbSubsystem climb;
+    private Timer leftCurrentTimer = new Timer();
+    private Timer rightCurrentTimer = new Timer();
+
+    public ZeroClimbersCommand(ClimbSubsystem climb) {
+        this.climb = climb;
+        addRequirements(climb);
+    }
+
+    @Override
+    public void initialize() {
+        leftCurrentTimer.stop();
+        leftCurrentTimer.reset();
+
+        rightCurrentTimer.stop();
+        rightCurrentTimer.reset();
+
+        climb.leftClimber.enableWinchSoftLimitReverse(false);
+        climb.rightClimber.enableWinchSoftLimitReverse(false);
+    }
+
+    private static void zeroSide(double power, Climber climber, Timer currentTimer) {
+        if (climber.winch.getOutputCurrent()
+                >= MoPrefs.climberZeroCurrentCutoff.get().in(Units.Amps)) {
+            if (currentTimer.hasElapsed(MoPrefs.climberZeroTimeCutoff.get().in(Units.Seconds))) {
+                climber.encoder.setPosition(Units.Centimeters.of(0));
+                climber.hasZero.setBoolean(true);
+            }
+        } else {
+            currentTimer.restart();
+        }
+
+        climber.runWinch(-Math.abs(MoPrefs.intakeZeroPwr.get()));
+    }
+
+    @Override
+    public void execute() {
+        zeroSide(MoPrefs.climberZeroPwr.get(), climb.leftClimber, leftCurrentTimer);
+        zeroSide(MoPrefs.climberZeroPwr.get(), climb.rightClimber, rightCurrentTimer);
+    }
```

Comment:
Once one side has a zero, we should not be commanding power to it anymore, even if the other side hasn't zeroed yet. 

This isn't a problem in the ZeroIntakeCommand because once we have a zero, the command exits. But here, one side can zero before the other is finished, but we'll keep commanding `climber.runWinch(-Math.abs(MoPrefs.intakeZeroPwr.get()));` to both sides.

One solution is as follows:
1. In `initialize()`, set `climber.hasZero` to false for both sides.
2. In `zeroSide()`, check if `climber.hasZero` is true. If so, command zero power. Otherwise, command `intakeZeroPwr`.